.travis.yml

```diff
@@ -16,7 +16,16 @@ env:
 script: test/k8s-local-cluster-test/run-test -v $KUBE_VERSION
 
 matrix:
-  include: 
+  include:
+    - stage: Test
+      language: go
+      go:
+        - 1.13
+      before_script:
+        - go get -t -v ./...
+      script: go test ./... -coverprofile=coverage.txt -covermode=atomic
```

Comment:
this should ideally go in the Makefile so we can just run `make unit` and then add as a dependent target in `make test`. The coverage.txt output should go to the build dir so that we don't accidentally check it in. The codecov bash script looks like it checks subdirs from the root so it should still find it or you can pass in the abs path to the codecov script.

Reply:
Yes, that's a best practice, Brandon. Have a `make unit-test` target (or similar) and run `make unit-test` from the CI job.

Example from VPC CNI:

https://github.com/aws/amazon-vpc-cni-k8s/blob/8a59da0e3e29bc06a9d419f212cde0d4897cd6c6/Makefile#L57-L58

https://github.com/aws/amazon-vpc-cni-k8s/blob/8a59da0e3e29bc06a9d419f212cde0d4897cd6c6/.circleci/config.yml#L18